Sources/Core/Presenter/Channel/ChannelPresenter.swift

```diff
@@ -50,6 +50,8 @@ public final class ChannelPresenter: Presenter {
     public var channel: Channel { channelAtomic.get() }
     /// A parent message for replies.
     public let parentMessage: Message?
+    /// Checks if the presenter is in a thread.
+    public var isThread: Bool { parentMessage != nil }
```

Comment:
I suggest renaming to `isThreadReply` or `isReply`

Reply:
This is for `ChannelPresenter` and it handles replies of a parent message in a thread.